maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/mapper/HardshipReviewMapperTest.java

```diff
@@ -52,7 +52,7 @@ void givenWorkflowRequestWithMagsHardship_whenWorkflowRequestToPerformHardshipRe
         WorkflowRequest workflowRequest = TestModelDataBuilder.buildWorkflowRequestWithHardship(CourtType.MAGISTRATE);
 
         ApiPerformHardshipRequest apiPerformHardshipRequest =
-                hardshipMapper.workflowRequestToPerformHardshipRequest(workflowRequest);
+                hardshipMapper.workflowRequestToPerformHardshipRequest(workflowRequest, false);
```

Comment:
Would it be worth having a scenario to cover when isCreate is passed through as both true and false to verify it gets the total annual disposable income from the right object?

Reply:
Good shout, updated tests to cover isCreate as true.